src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/GetExtendedQueryTagHandler.cs

```diff
@@ -7,15 +7,20 @@
 using System.Threading.Tasks;
 using EnsureThat;
 using MediatR;
+using Microsoft.Health.Core.Features.Security.Authorization;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Features.Common;
+using Microsoft.Health.Dicom.Core.Features.Security;
 using Microsoft.Health.Dicom.Core.Messages.ExtendedQueryTag;
 
 namespace Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag
 {
-    public class GetExtendedQueryTagHandler : IRequestHandler<GetExtendedQueryTagRequest, GetExtendedQueryTagResponse>
+    public class GetExtendedQueryTagHandler : BaseHandler, IRequestHandler<GetExtendedQueryTagRequest, GetExtendedQueryTagResponse>
```

Comment:
What about AddExtendedQueryTagHandler?

Reply:
it has been done before https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/AddExtendedQueryTagHandler.cs